.github/workflows/lint-and-test.yml

```diff
@@ -94,7 +94,7 @@ jobs:
         run: |
           kubectl create namespace redis
           helm repo add bitnami https://charts.bitnami.com/bitnami
-          helm install redis bitnami/redis --wait --namespace redis --set auth.password=argocd --set architecture=standalone
+          helm install redis bitnami/redis --wait --namespace redis --set auth.password=argocd --set architecture=standalone --set global.image.repository=bitnamilegacy/redis
```

Comment:
will this override the existing image here - `helm install redis bitnami/redis`?

Also, shouldn't we be using official redis image instead as discussed in meeting?